src/cpp/session/modules/SessionConnections.R

```diff
@@ -552,18 +559,18 @@ options(
             sep = "")
       }
 
-      licenseFile <- file.path(dirname(currentDriver$value), "license.lock")
+      licenseFile <- file.path(dirname(currentDriver), "license.lock")
 
       iconData <- .Call("rs_connectionIcon", driverId)
       if (nchar(iconData) == 0)
          iconData <- .Call("rs_connectionIcon", "ODBC")
 
-      hasInstaller <- identical(driverInstaller$value, "RStudio")
+      hasInstaller <- identical(driverInstaller, "RStudio")
       warningMessage <- NULL
 
       if (hasInstaller) {
          installerVersion <- .rs.connectionInstallerInfo(driverId)$version
-
+         currentVersion <- drivers[drivers$attribute == "Version" & drivers$name == driver, ]$value
          currentVersion <- drivers[drivers$attribute == "Version" & drivers$name == driver, ]
```

Comment:
I think only one of these assignments is intentional -- I believe we want the new addition only (line 573)?